.github/dockerfiles/ov_build/debian_10/Dockerfile

```diff
@@ -0,0 +1,86 @@
+FROM openvinogithubactions.azurecr.io/dockerio/library/debian:10.13
+
+USER root
+
+# APT configuration
+RUN echo 'Acquire::Retries "10";' > /etc/apt/apt.conf && \
+    echo 'APT::Get::Assume-Yes "true";' >> /etc/apt/apt.conf && \
+    echo 'APT::Get::Fix-Broken "true";' >> /etc/apt/apt.conf && \
+    echo 'APT::Get::no-install-recommends "true";' >> /etc/apt/apt.conf
+
+ENV DEBIAN_FRONTEND="noninteractive" \
+    TZ="Europe/London"
+
+RUN apt-get update && \
+    apt-get install \
+        software-properties-common \
+        curl \
+        git \
+        gpg-agent \
+        tzdata \
+        # Pythons \
+        python3 \
+        python3-pip \
+        python3-dev \
+        python3-venv \
+        python3-distutils \
+        build-essential \
+        libffi-dev \
+        libgdbm-dev \
+        libc6-dev \
+        libssl-dev \
+        zlib1g-dev \
+        libbz2-dev \
+        libreadline-dev \
+        libsqlite3-dev \
+        libncurses5-dev \
+        libncursesw5-dev \
+        xz-utils \
+        tk-dev \
+        libxml2-dev \
+        libxmlsec1-dev \
+        liblzma-dev \
```

Comment:
looks like a lot of dependencies above are not required. Let's keep minimal t of packages as for Ubuntu 20-24 x64

Reply:
I tried removing some of them and while Python installed and (seemingly) worked correctly, the build repeatedly failed with errors like: `/usr/local/include/python3.11/pyport.h:601:2: error: #error "LONG_BIT definition appears wrong for platform (bad gcc/glibc config?)."`

Reply:
ok, let's keep as is to save our time.